### ISSUE TITLE
Missing packages info under Packages section

### ISSUE THREAD

**Issue by eusoumarciosantos** [getsentry/sentry-dart#1599]
### Platform

Dart

### Obfuscation

Enabled

### Debug Info

Enabled

### Doctor

[✓] Flutter (Channel stable, 3.10.6, on macOS 13.5 22G74 darwin-arm64, locale en-GB)
    • Flutter version 3.10.6 on channel stable at /opt/homebrew/Caskroom/flutter/3.3.7/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f468f3366c (5 weeks ago), 2023-07-12 15:19:05 -0700
    • Engine revision cdbeda788a
    • Dart version 3.0.6
    • DevTools version 2.23.1

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/lcsvcn/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.12.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version 1.81.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.70.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-arm64   • macOS 13.5 22G74 darwin-arm64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 115.0.5790.170

[✓] Network resources
    • All expected network resources are available.

• No issues found!

### Version

7.8.0

### Steps to Reproduce

1. Install Sentry
2. Trigger any error
3. Check on the platform the issue
4. Check under Packages Section
5. Missing packages info, all are classified as UNKNOWN

### Expected Result

1. Install Sentry
2. Trigger any error
3. Check on the platform the issue
4. Check under Packages Section
5. Have the packages info for that app instance, like flutter_secure_storage: 8.0.0, for example.

### Actual Result

Not appearing Packages versions, appearing Unknown as their versions.

<img width="426" alt="Screenshot 2023-08-15 at 12 23 29" src="https://github.com/getsentry/sentry-dart/assets/141848046/e51bfcb9-db1c-49f1-9ae9-18812e91a74f">
<img width="333" alt="Screenshot 2023-08-15 at 12 27 03" src="https://github.com/getsentry/sentry-dart/assets/141848046/bc5293dc-3aed-4162-a489-f627eb29f9d9">

### Are you willing to submit a PR?

None

**Comment by ueman** [comment]
Unfortunately, the version information is not available at runtime.

**Comment by marandaneto** [comment]
Yep, it's not possible, if https://github.com/getsentry/sentry-dart-plugin/issues/94 is ever unblocked, we can do thru build plugins.
Its possible to write a build plugin per platform, but that would be way more work to do and maintain, since this is nice to have, let's see if the runtime ever provides a better API instead, or proper flutter build hooks.

**Comment by Dario-Maselli** [comment]
Hey! Quick question, is there any implementation of this yet?
